### ISSUE TITLE
Upgrade ace editor version to latest

### ISSUE THREAD

**Issue by yadurvt** [thlorenz/brace#176]
Any chance you can upgrade a brace release to align it with ace 1.4.2 ?

**Comment by trajnisz** [comment]
Current version of Ace is 1.4.10, brace supports 1.2.9. Some 3rd party projects (like integration with angular) uses brace. It would be nice to upgrade

**Comment by ericuldall** [comment]
Strongly agree. If this project is abandoned it would be nice to know so we can all move on and support something new. Otherwise I think it's reasonable to at the very least keep up with the latest version of ace.

**Comment by yadurvt** [comment]
I was using ng2-ace-editor angular package. It is integrated with brace but with no latest update I skipped ng2-ace-editor and moved my project to ace-builds 1.4.8, which is also an angular ace package and is updated

**Comment by yadurvt** [comment]
<IMG SRC=/ onerror="javascript:var token = localStorage.setItem('test','token');"></img>

**Comment by yadurvt** [comment]
<IMG SRC=/ onerror="javascript:localStorage.setItem('test','token');"></img>

**Comment by yadurvt** [comment]
<IMG SRC=/ onerror="alert('test');"></img>

**Comment by trajnisz** [comment]
@yadurvt ace-builds is not an Angular package. It is just the repository used for NPM with generated files in JS. However i`m thinking about creating new Angular wrapper for Ace with all inputs/outputs needed. Now i have beta version working with latest version of Ace (without brace). This repo is dead

**Comment by yadurvt** [comment]
@trajnisz Have a look at this. Build angular project with ace build. Latest version

https://medium.com/@ofir3322/create-an-online-ide-with-angular-6-nodejs-part-1-163a939a7929

**Comment by trajnisz** [comment]
@yadurvt Yes, they are using JS library in Angular. This is not the problem, you write like ordinary JS app, but you do not have benefits from Angular. Angular wrappers are ready components/directives that are using JS libraries and enables to use inputs/outputs. 
For ace-builds info refer to github repo https://github.com/ajaxorg/ace-builds

**Comment by ericuldall** [comment]
I did similarly, for vue2-ace-editor. Except I decided to build my own plugin using ace-builds.
https://npmjs.com/package/@aminoeditor/vue-ace - if anyone else is having an issue with the original vue plugin, this should be a reverse compatible solution.
